### ISSUE TITLE
KTO全参数微调训练 logits/chosen数量级很大，负几百万，kl是0

### ISSUE THREAD

**Issue by yaojianchao5-droid** [modelscope/ms-swift#7342]
<img width="303" height="228" alt="Image" src="https://github.com/user-attachments/assets/460e5eb9-90ec-4108-b20c-5c747a239bd8" />

**Comment by hjh0119** [comment]
提供一下完整的训练脚本和环境信息

**Comment by yaojianchao5-droid** [comment]
> 提供完整的训练脚本和环境信息

<img width="360" height="246" alt="Image" src="https://github.com/user-attachments/assets/4941fb4b-a60d-4935-a061-0846f71a6ba3" />   训练环境是swift3.11 A100   
KTO训练数据由DPO训练数据转换。DPO训练正常，数据集应该是没问题的。
